.vscode/extensions.json

```diff
@@ -1,13 +1,7 @@
 {
-  // See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
```

Comment:
Ideally these unrelated changes are split into their own PRs, so each PR only does what it says in the PR description.
But I'm fine for this one since this is harmless, just something to keep in mind.